### PR TITLE
Fixing tests for LargeBarrelAnalysis

### PR DESCRIPTION
**Pull request by grey277**
Add syslink to unitTestData folder in LargeBarrelAnalysis' test folder, small changes to EventCategorizerTools to make tests work. SignalFinderToolsTests are still not working, null objects are returned from ParamBank.

### CODE REVIEW

File: LargeBarrelAnalysis/EventCategorizerToolsTest.cpp [hunk 1]
```diff
@@ -79,17 +79,17 @@ BOOST_AUTO_TEST_CASE(pointAtCenter)
...
+  firstHit.setTime(1333 / 2.0);
+  firstHit.setPos(0, 45, 0);
   JPetBarrelSlot firstSlot(1, true, "first", 90, 1);
   firstHit.setBarrelSlot( firstSlot );
-  
+
   JPetHit secondHit;
-  secondHit.setTime(1667/2);
-  secondHit.setPos(0,-45,0);
+  secondHit.setTime(1667 / 2);
```
Comment (alekgajos) [inline LargeBarrelAnalysis/EventCategorizerToolsTest.cpp]: Is this an integer division on purpose? It probably does not make a difference in the test, but just for correctness we could make it explicitly floating-point.

File: LargeBarrelAnalysis/EventCategorizerToolsTest.cpp [hunk 2]
```diff
@@ -100,17 +100,17 @@ BOOST_AUTO_TEST_CASE(pointAt0x_5y_0z)
...
+  firstHit.setTime(1333 / 2.0);
+  firstHit.setPos(0, -45, 0);
   JPetBarrelSlot firstSlot(1, true, "first", 270, 1);
   firstHit.setBarrelSlot( firstSlot );
-  
+
   JPetHit secondHit;
-  secondHit.setTime(1667/2);
-  secondHit.setPos(0,45,0);
+  secondHit.setTime(1667 / 2);
```
Comment (alekgajos) [inline LargeBarrelAnalysis/EventCategorizerToolsTest.cpp]: same here

**Comment by alekgajos** [comment]
The PR is OK, but it probably duplicates PR #61, where the tests are also extended.

**Comment by wkrzemien** [comment]
Duplicated of  PR #61.

**Pull request closed without merging**